### DISCUSSION TITLE
useFieldArray for  React native?

### DISCUSSION THREAD

**Discussion by ashfaq1994** [react-hook-form/react-hook-form#4790] [Q&A]
Hello!,
Looking for useFieldArray in react native for Android. There is not example for React native, I can see for only React Web.

Please let me know where do I find the exactly example for react native.

Thanks

**Comment by bluebill1049** [comment]
same usage, just replace with `Controller` for each inputs.

**Reply by jetaggart** [reply]
I'm having a pretty difficult time figuring out how to do this. I feel like I'm close but random things break. For example, I have it all working with `Controller` but now if I remove, then append again, it injects multiple items with undefined ids. Is there any chance you could provide a simple example? I feel like I'm missing something small but important. Thank you!

**Reply by lundn** [reply]
@jetaggart Could you provide an Expo Snack, will be easier to help out.
